person.rb

```diff
@@ -1,21 +1,30 @@
 class Person
-  attr_accessor :name, :age
-  attr_reader :id
+  require_relative 'nameable'
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
-    @id = Random.rand(1...1000)
-    @name = name
-    @age = age
-    @parent_permission = parent_permission
-  end
+  class Person < Nameable
+    attr_accessor :name, :age
```

Comment:
- [ ] Hi putting a class Person inside of another class Person is not very ideal and may cause further issues. I'll recommend removing the first Class Person since you need the one inheriting from Numerable

Reply:
> * [x]  Hi putting a class Person inside of another class Person is not very ideal and may cause further issues. I'll recommend removing the first Class Person since you need the one inheriting from Numerable

Thank you for your suggestion.
Removed the same.